doc/source/cluster/kubernetes/user-guides/kuberay-gcs-ft.md

```diff
@@ -0,0 +1,261 @@
+(kuberay-gcs-ft)=
+# GCS fault tolerance in KubeRay
+
+Global Control Service (GCS) manages cluster-level metadata.
```

Comment:
Is this a component of KubeRay? Can we say, "The Global Control Service (GCS) is a component of KubeRay that manages cluster-level metadata?" Or is that obvious?

Reply:
It is a Ray Core component.